sndev

```diff
@@ -9,7 +9,11 @@ docker__compose() {
     exit 0
   fi
 
-  CURRENT_UID=$(id -u) CURRENT_GID=$(id -g) command docker compose --env-file .env.development "$@"
+  if [ -z "$COMPOSE_PROFILES" ]; then
+    COMPOSE_PROFILES="images,search,payments"
+  fi
+
+  CURRENT_UID=$(id -u) CURRENT_GID=$(id -g) COMPOSE_PROFILES=$COMPOSE_PROFILES command docker compose --env-file .env.development "$@"
```

Comment:
How does `minimal` work here? I don't see `minimal` referenced in the changes to `docker-compose.yml`, so I assume that means any services with no profiles are started, and since no services are tagged with `minimal`, then you just get those without profiles at all?

Reply:
Exactly. It's just a way to communicate that you don't want all or any other profile. Because I want the default to be all.